addons/website_forum/views/website_forum.xml

```diff
@@ -818,7 +871,7 @@
             <h5 class="mb-1 text-muted" t-esc="post.parent_id.name"/>
             <div t-field="post.parent_id.content" class="o_wforum_post_content text-muted oe_no_empty"/>
         </article>
-        <form t-attf-action="/forum/#{slug(forum)}/post/#{slug(post)}/save" method="post" role="form" class="tag_text js_website_submit_form o_wforum_readable">
```

Comment:
Why removing the old class ? We still want the spinner effect which is there for all website forms if possible. If not valid on submit, remove the spinner and re-enable the button, we'll improve this system later.

Reply:
Because now, when the requirements aren't validated, we want the spin disappear. And having both classes add 2 spin and deletes one. I can only delete in the new one, but I don't know if we can control the order of which class is trigger first. If it's not the case, the deletion could trigger first and we will still have the spinner even if the form is not valid.